app/controllers/avo/associations_controller.rb

```diff
@@ -188,5 +188,9 @@ def join_record
       reflection.through_reflection.klass.find_by(source_foreign_key => @attachment_record.id,
         through_foreign_key => @record.id)
     end
+
+    def has_many_reflection?
+      reflection_class.in? ["HasManyReflection", "HasAndBelongsToManyReflection"]
```

Comment:
I noticed that above some mentions are classes and these are strings.
Is there anything we can do to make them the same or do we have to know when to call as a string and as a class?

Reply:
Makes sense. There is no reason to use `class.name.demodulize.to_s` and check against a string.